Makefile

```diff
@@ -379,19 +379,9 @@ golangci_version=v1.51.2
 lint:
 	@echo "--> Running linter"
 	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(golangci_version)
-	@$(golangci_lint_cmd) run --timeout=10m
+	@sh ./scripts/go-lint-all.sh
 
-lint-fix:
-	@echo "--> Running linter"
-	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(golangci_version)
-	@$(golangci_lint_cmd) run --fix --out-format=tab --issues-exit-code=0
-
-.PHONY: lint lint-fix
-
-format:
-	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(golangci_version)
-	$(golangci_lint_cmd) run --fix
```

Comment:
the script seems to assume the fix every time, is there a need to have them separate? also we can make it flag based with the script as well.

Reply:
Thought of that in my initial message, but then I don't think there is a use case where we'd want to simply see the issues and not fix them.

Reply:
that is true, we can skip then